test/DebuggerCommon/ES6_proto_invalidation.js.dbg.baseline

```diff
@@ -459,13 +465,6 @@
               "caller": "object null",
               "arguments": "object null"
             },
-            "indexOf": {
-              "#__proto__": "function <large string>",
-              "length": "number 1",
-              "name": "string indexOf",
-              "caller": "object null",
-              "arguments": "object null"

```

Comment:
Status : 'caller' and 'arguments' were not set anymore. It's fixed. The 'name' property was not set correctly. It's fixed. The 'prototype' is under investigation

---
In reply to: [114233872](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r114233872) [](ancestors = 114233872)